Description: Avoid "RuntimeError: Settings already configured."
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 planet-venus (0~git9de2109-1) UNRELEASED; urgency=low
 .
   * New upstream version.
Author: Olivier Berger <obergix@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- planet-venus-0~git9de2109.orig/planet/shell/dj.py
+++ planet-venus-0~git9de2109/planet/shell/dj.py
@@ -19,13 +19,13 @@ def run(script, doc, output_file=None, o
     # I need to re-import the settings at every call because I have to 
     # set the TEMPLATE_DIRS variable programmatically
     from django.conf import settings
-    settings._wrapped=None
+    #settings._wrapped=None
     try:
         settings.configure(
             DEBUG=True, TEMPLATE_DEBUG=True, 
             TEMPLATE_DIRS=(os.path.dirname(script),)
             )
-    except EnvironmentError:
+    except RuntimeError:
         pass
     from django.template import Context
     from django.template.loader import get_template
